x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/editor/formula_editor.tsx

```diff
@@ -726,36 +732,39 @@ export function FormulaEditor({
                         <EuiButtonEmpty
                           color={errorCount ? 'danger' : 'warning'}
                           className="lnsFormula__editorError"
+                          iconType="alert"
+                          size="xs"
                           onClick={() => {
                             setIsWarningOpen(!isWarningOpen);
                           }}
                         >
```

Comment:
Can we add a prop of `flush="right"` to this `EuiButtonEmpty` to align correctly?